cli/reporter/cli_reporter.go

```diff
@@ -101,6 +101,13 @@ func NewReporter(format Format, incognito bool) *Reporter {
 	}
 }
 
+// This allows the packer-plugin-cnspec to set the output writer directly
```

Comment:
nit: can we update the comment to just describe what this func does? we shouldnt reference which component needs that as the comment might become outdated over time

Reply:
also consider adding this to the `NewReporter` func instead, would be nice to get out the hardcoded `os.Stdout` there

Reply:
But, isn't `os.Stdout` the correct default for the CLI reporter?